server/products.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const db = require('APP/db')
+const Product = db.model('products')
+
+module.exports = require('express').Router()
```

Comment:
How do you all feel about adding a descriptive comment for each route, to reduce confusion in the future?